build.gradle

```diff
@@ -98,7 +98,7 @@ dependencies {
         dropwizardVersion = '3.1.2'
     }
 
-    pgsql 'org.postgresql:postgresql:9.4.1207'
+    pgsql 'org.postgresql:postgresql:42.1.1'
```

Comment:
@lmontrieux on the line 124 we have entry:
+    compile 'org.postgresql:postgresql:42.1.1'
looks the same as on the line 101